test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -163,6 +164,22 @@ public static ClientConfig buildClientConfig(URI controllerUri) {
 
     }
 
+    public static Map<String, Object> buildPatchedPravegaClusterSpec(String replicas, int replicaCount, String component, String namespace, String name, String kind) {
+
+        final Map<String, Object> componentSpec = ImmutableMap.<String, Object>builder()
+                .put(replicas, replicaCount)
+                .build();
+
+        return ImmutableMap.<String, Object>builder()
+                .put("apiVersion", "pravega.pravega.io/v1alpha1")
```

Comment:
`kubernetes.AbstractService#CUSTOM_RESOURCE_GROUP_PRAVEGA` and `kubernetes.AbstractService#CUSTOM_RESOURCE_VERSION_PRAVEGA` refer to the above string, we should reuse it for easier maintainability.
We could move this to `io.pravega.test.system.framework.services.kubernetes.AbstractService`.

Reply:
done